vm/src/exceptions.rs

```diff
@@ -361,6 +358,7 @@ pub struct ExceptionZoo {
     pub arithmetic_error: PyClassRef,
```

Comment:
Do you want to reorder this too to be in line with exceptions.txt?

Reply:
I am neutral, but will do if you want

Reply:
I think it would be good to have a consistent order for them across the 4 places we list them (ExceptionZoo struct definition, putting all of the exceptions into variables, constructing ExceptionZoo, adding them to `builtins`)

Reply:
fixed it